src/subscription_manager/repolib.py

```diff
@@ -51,10 +51,10 @@
 
 class YumPluginManager(object):
     """
-    Instance of this class is used for automatic enabling of yum plugins.
+    Instance of this class is used for automatic enabling of dnf plugins
+    (formerly for yum plugins, hence the name).
     """
 
-    YUM_PLUGIN_DIR = '/etc/yum/pluginconf.d'
```

Comment:
Question: Doesn't `dnf` still read this directory due to backward compatibility?

Reply:
The only occurrence of that I found was in the spec file of dnf; in particular, it seems that the yum location is created as symlink to the dnf location in case the "full yum compatibility" is enabled (I guess what's done in RHEL):
https://github.com/rpm-software-management/dnf/blob/e7b29753dce94769d30f92e671f55863985d24f0/dnf.spec#L229-L234